advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx

```diff
@@ -0,0 +1,185 @@
+---
+
+How to backups and restore from EDB Postgres Advanced Server using Commvault Backup & Recovery.
+
+## Using Commvault Backup & Recovery
+
+Commvault provides two methods of taking the backup from EDB Postgres Advanced Server and restoring it.
+1. DumpBasedBackupSet Backup and Restore
+2. FSBasedBackupSet Backup and Restore
+
+### DumpBasedBackupSet Backup and Restore
+
+Dump based backup uses the pg_dump Utility to take the backup.
+
+#### Taking DumpBasedBackupSet Backup 
+
+1. Open Commvault Backup & Recovery's Command Centre and from the navigation pane, go to Protect > Databases.
+
+<p align="center">
+  <img  alt="Instances Page" src="Images/Dumpbackup1.png">
+</p>
+
+2. Click on the required instance.
+3. In the Backup sets section, click on DumpBasedBackupSet backup set.
+
+<p align="center">
+  <img  alt="Select Backup Set" src="Images/Dumpbackup2.png">
+</p>
+
+4. In the Database groups section, click the database group that you want to back up. In this case its default.
+
+<p align="center">
+  <img  alt="Select Database Group" src="Images/Dumpbackup3.png">
+</p>
+
+5. In the Backup section, click Back up now.
+
+<p align="center">
+  <img  alt="Backup Section" src="Images/Dumpbackup4.png">
+</p>
+
+6. Select Full in the Select Backup Level Screen.
+
+<p align="center">
+  <img  alt="Backup Level Screen" src="Images/Dumpbackup5.png">
+</p>
+
+7. A Job will be created to take the backup.
+
+<p align="center">
+  <img  alt="Backup Job" src="Images/Dumpbackup6.png">
+</p>
+
+8. Once the Backup Job is completed then its status will be changed to Completed.
+
+<p align="center">
+  <img  alt="Backup Job" src="Images/Dumpbackup7.png">
+</p>
+
+#### Restoring DumpBasedBackupSet Backup 
+
+DumpBased backupset can be used to restore the individual databases.
+
+1. From the navigation pane, go to Protect > Databases.
+2. Click the instance that you want to restore.
+3. In the Recovery points calendar, select DumpBasedBackupSet.
+
+<p align="center">
+  <img  alt="Recovery Points Calendar" src="Images/Dumprestore1.png">
+</p>
+
+4. Select a date from the calendar, and then click Restore.
+5. Backup Content Screen will display the databases to be restored, select the required database to restore or select all of them to restore all.
+
+<p align="center">
+  <img  alt="Backup Content Screen" src="Images/Dumprestore2.png">
+</p>
+
+6. Click Restore.
+7. From the Restore Options Screen, select the Destination Server and Destination Instance and click on submit.
+
+<p align="center">
+  <img  alt="Restore Options Screen" src="Images/Dumprestore3.png">
+</p>
+
+8. A Job will be created to restore the backup.
+
+<p align="center">
+  <img  alt="Restore Job" src="Images/Dumprestore4.png">
+</p>
+
+<p align="center">
+  <img  alt="Restore Job" src="Images/Dumprestore5.png">
+</p>
+
+9. Once the Restore is completed successfully, login to the EDB Postgres Advanced Server Instance and check that restore operation recovered the data.
```

Comment:
login to the EDB database and check that the restore operation recovered the data.  In our example below we connected to an EDB Postgres Advanced Server.